content/doc/book/managing/script-approval.adoc

```diff
@@ -0,0 +1,140 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:hide-uri-scheme:
+:imagesdir: /doc/book/resources
+
+= In-process Script Approval
+
+Jenkins, and a number of plugins, allow users to execute
+Groovy scripts _in_ Jenkins. These scripting capabilities are provided by:
+
+* <<script-console#, Script Console>>.
+* <<../pipeline#, Jenkins Pipeline>>.
+* The plugin:email-ext[Extended Email plugin].
+* The plugin:groovy[Groovy plugin] - when using the "Execute system Groovy
+  script" step.
+* The plugin:job-dsl[JobDSL plugin] as of version 1.60 and later.
+
+To protect Jenkins from execution of malicious scripts, these plugins execute
+user-provided scripts in a <<groovy-sandbox>> that limits what internal APIs
+are accessible.  Administrators can then use the "In-process Script Approval"
+page, provided by the plugin:script-security[Script Security plugin], to manage
+which unsafe methods, if any, should be allowed in the Jenkins environment.
+
+image::managing/manage-inprocess-script-approval.png["Entering the In-process Script Approval configuration", role=center]
+
+== Getting Started
+
+The plugin:script-security[Script Security plugin] is installed automatically
+by the
+link:../../getting-started/installing#post-install-setup-wizard[Post-install Setup Wizard],
+although initially no additional scripts or operations are approved for use.
+
+[IMPORTANT]
+====
+Older versions of this plugin may not be safe to use. Please review the
+following advisory:
+link:/security/advisory/2016-04-11/[Groovy sandbox protection incomplete]
```

Comment:
🐜 Please link to Security Advisories instead of the particular advisory. If there are new security advisories for this plugin, the admin may be confused.

Ideally we need to support filtering of user advisories by plugin somehow. Maybe by the page tags (the most simple way to implement I'd guess). CC @daniel-beck